content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,43 +2,78 @@
 title: Google Cloud Run
 kind: documentation
 further_reading:
-    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
-      tag: 'Blog'
-      text: 'Collect traces, logs, and custom metrics from GCR services'
+
+- link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+  tag: 'Blog'
+  text: 'Collect traces, logs, and custom metrics from GCR services'
+
 ---
 
 ## Overview
-Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution, now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
+
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog
+provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution,
+now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom
+metrics, and direct log collection.
 
   <div class="alert alert-warning">This feature is in public beta. You can provide feedback through a <a href="https://forms.gle/HSiDGnTPvDvbzDAQA">feedback form</a>, or through your standard support channels. During the beta period, Cloud Run monitoring and APM tracing are available without a direct cost. Existing APM customers may incur increased span ingestion and volume costs. </div>
 
-## Tracing and custom metrics
-### Build your container
+## Getting started
+
+### Prerequisites
+
+Make sure you have a [Datadog API Key][10] and are using a programming
+language [supported by a Datadog tracing library][2]. To deploy a sample application in one click, without the need to
+follow the rest of the steps, you can use the Cloud Run Button of [these examples][12].
+
+### Build your container with Datadog instrumentation
+
+Cloud Run requires applications to be packaged in a container. From the application root folder use one
+of the following methods.
+
+#### Using Datadog buildpack
 
-If you are using a Dockerfile to build your application, complete the following:
+[`Pack Buildpacks`][4] provide a convenient way to package your container without using a Dockerfile. This example
+will use the GCP container registry
+and [Datadog serverless buildpack](https://registry.hub.docker.com/r/datadog/serverless-buildpack).
+Build your application by running the following:
 
-1. Instrument your application with a [supported Datadog tracing library][2]
+   ```shell
+   pack build --builder=gcr.io/buildpacks/builder \
+   --buildpack from=builder \
+   --buildpack datadog/serverless-buildpack:beta4 \
```

Comment:
the latest is beta6 (actually @IvanTopolcic just found a problem with it, so don't update it), but @maxday we need to think about how to keep this up to date as well.

Reply:
I think we should ask to get the latest version from the link, it's in beta so there might be issue, but once it's stable it should be ok.
Or we could fallback to latest/default